### PR TITLE
[Port-ish]Desk bells for everyone!

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

**CURRENTLY ON HOLD AS THERE HASN'T BEEN ANY MAP UPDATE, AS I AM WAITING FOR THE GREP PILE PORT TO BE MERGED**

- [x] Actually add the sound
- [x] Add bells on all the maps.
- [x] ~~Port https://github.com/tgstation/tgstation/pull/67821/files~~ Cannot due to a planned rideable refactoring
- [ ] figure out a way to add signaling

This is a partialport of https://github.com/tgstation/tgstation/pull/67199, but the code has been strongly adpated for Beestation, alongside a new sprite for the hyper bell giving a distinct Fancy golden style.

## Why It's Good For The Game

The lack of non-destructive annoyance in the station is bad, and this is an INCREDIBLY annoying way to do so

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

_I.O.Y. a video with sound_ , for now have the sprites

immagine

</details>

## Changelog
:cl:
add: Adds a desk bell, so you can let the service industry know how much you truly care about their hearing, alongside a new distinct sound for it!.
/:cl:

### CODE REVIEW

File: code/modules/paperwork/desk_bell.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		return TRUE
+	if(!ring_bell(user))
+		to_chat(user,"<span class='notice'>[src] is silent. Some idiot broke it.</span>")
+	if(ring_cooldown_length)
+		COOLDOWN_START(src, ring_cooldown, ring_cooldown_length)
+	return TRUE
+
+/obj/structure/desk_bell/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	times_rang += I.force
```
Comment (PowerfulBacon) [inline code/modules/paperwork/desk_bell.dm:39]: ~~Stuff that does 0 damage should still increment it~~ (nvm it does in ring_bell), but won't they just cause integrity damage directly and break it anyway?

Comment (PestoVerde322) [reply code/modules/paperwork/desk_bell.dm:39]: I think it's intended, as it might be consider as "tampering" with it

**Comment by CydiaLamiales** [comment]
Include a video with the bell being abused for ding-ding-ding spam
pretty please

**Comment by Tsar-Salat** [comment]
> ding-ding-ding spam
> pretty please

add hitting it too many time on harm intent has a chance to embed it in your arm lmao

**Comment by itsmeow** [comment]
bump

**Comment by PestoVerde322** [comment]
I'll update it asap, waiting for #8401 to merge

**Review by itsmeow — APPROVED** [review]
Lovely, hard to review the mapping changes but the diffs look fine at a glance

**Comment by CydiaLamiales** [comment]
Where sound, I wanna see a video of a bell being absolutely abused.

**Pull request merged**